src/App.jsx

```diff
@@ -1,9 +1,93 @@
 import './App.css'
+import Button from "./componets/button/button.jsx";
+import bag1 from "./assets/bag_1.png"
+import bag2 from "./assets/bag_2.png"
+import bag3 from "./assets/bag_3.png"
+import bag4 from "./assets/bag_4.png"
+import BagItem from "./componets/bagItem/bagItem.jsx"
+import brand from "./assets/brand.png"
+import ourStory from "./assets/our_story.png"
+import Tile from "./componets/tile/tile.jsx"
 
 function App() {
   return (
-      <h1>Handbags & Purses</h1>
-  )
+
+  <>
+
+    <h1>Handbags & Purses</h1>
+
+<nav>
+  <Button
+  buttonText = "to the collection"
+  buttonType="button"
+  disabledButton={false}
+  />
+  <Button
+      buttonText = "shop all bags"
+      buttonType="button"
+      disabledButton={false}
+  />
+  <Button
+      buttonText = "pre-orders"
+      buttonType="button"
+      disabledButton={true}
+  />
+</nav>
+
+    <main>
+
+      <BagItem
+          bagLabel="Best seller"
+          bagImg = {bag1}
+          bagDescription = "handy"
+          bagPrice = "€400,-"
```

Comment:
Bij de bagPrice hoef je alleen het getal te vermelden, als je in de component al €{price},- hebt vermeld.